modules/compliance-operator-cli-installation.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-installation.adoc
+
+[id="installing-compliance-operator-cli_{context}"]
+= Installing the Compliance Operator using the CLI
+
+.Prerequisites
+
+* You must have `admin` privileges.
+
+.Procedure
+
+. Create a namespace object YAML file by running:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-compliance
+----
+
+. Apply the file by running:
++
+[source,terminal]
+----
+$ oc apply -f namespace.yaml
+----
+
+. Create the `OperatorGroup` object YAML file by running:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: compliance-operator
+  namespace: openshift-compliance
+spec:
+  targetNamespaces:
+  - openshift-compliance
+----
+
+. Create the Subscription object YAML file by running:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: compliance-operator-sub
+  namespace: openshift-compliance
+spec:
+  installPlanApproval: Automatic
+  name: compliance-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
```

Comment:
for the subscription, there is an important field called "channel":
a. Run the following command to set the OpenShift Container Platform major and minor version as an environment variable, which is used as the channel value in the next step.
OC_VERSION=$(oc version -o yaml | grep openshiftVersion | grep -o '[0-9]*[.][0-9]*' | head -1)
b. Create the Subscription object YAML file by running:
----
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: compliance-operator-sub
  namespace: openshift-compliance
spec:
  channel: "${OC_VERSION}"
  installPlanApproval: Automatic
  name: compliance-operator
  source: redhat-operators
  sourceNamespace: openshift-marketplace
----

Reply:
It would be good if we mention channel while creating subscription but it is not mandatory. We can create subscription without mentioning `channel` and operator will get deployed using `defaultChannel`

> $ oc get packagemanifest compliance-operator -oyaml |grep defaultChannel
>   defaultChannel: "4.6"

Reply:
@pdhamdhe If works for 4.6. What if a 4.7 cluster. It will have 2 packagemanifests(one for 4.6 and the other one for 4.7). If so, for fresh install, it is better to recommend users to use 4.7.

Reply:
@xiaojiey Jfyi.. By default there will be only one packagemanifest for each ocp version for operator. So on 4.7 cluster, the compliance operator packagemanifest will only have `defaultChannel : "4.7" `. I was able to installed operator on both
the cluster i.e 4.6 & 4.7 using the subscription yaml file which is mentioned in documentation. Though, I am fine with both the methods.